src/main/java/io/openliberty/tools/intellij/util/LibertyGradleUtil.java

```diff
@@ -230,7 +230,10 @@ private static String getLocalGradleWrapperPath(String wrapperDir) throws Libert
         } catch (IOException e) {
             throw new LibertyException("Could not get canonical path for gradle wrapper file");
         }
-        return path;
+        String cmd = LibertyProjectUtil.includeEscapeToString(path);
+        if (SystemInfo.isWindows)
```

Comment:
Can we use ternary operator here similar to line no. 254, I felt the below code is more readable.

 String additionalCMD = SystemInfo.isWindows ? "cmd /K " : ""; // without it, a new terminal window is opened
 return additionalCMD + LibertyProjectUtil.includeEscapeToString(gradleExecutable.getAbsolutePath());

Reply:
@aparnamichael  ,Paul suggested this change. Initially, I made the same change you are suggesting now. Then Paul provided this suggestion as a fix. He already have raised a PR for changing the code in line number 254

https://github.com/OpenLiberty/liberty-tools-intellij/pull/871/files